source/scripting/Entity.cs

```diff
@@ -658,6 +656,16 @@ public Blip CurrentBlip
 				return null;
 			}
 		}
+		public Blip[] AttachedBlips
+		{
+			get
+			{
+				var allBlips = World.GetAllBlips();
+				var attachedBlips = new List<Blip>();
+				attachedBlips.AddRange(allBlips.Where(x => Function.Call<int>(Hash.GET_BLIP_INFO_ID_ENTITY_INDEX, x) == Handle));
+				return attachedBlips.ToArray();
```

Comment:
This could be simplified alot
return World.GetAllBlips().Where(x => Function.Call<int>(Hash.GET_BLIP_INFO_ID_ENTITY_INDEX, x) == Handle)).ToArray()